### ISSUE TITLE
The compiler worker does not log to file as the worker protocol expects.

### ISSUE THREAD

**Issue by hsyed** [pubref/rules_kotlin#55]
[see this](https://github.com/bazelbuild/intellij/pull/166#issuecomment-352234048). While investigating to see if the background workers add any performance gains after updating rules to follow the worker protocol protocol (see #54) I disovered that compiling erroneous code in error mode from a persistent workers leads to: 

```
---8<---8<--- Start of log, file at /private/var/tmp/_bazel_hassan/b35d318b00182129bd971b30e0d6ece1/bazel-workers/worker-4-KotlinCompile.log ---8<---8<---
(empty)
---8<---8<--- End of log ---8<---8<---
INFO: Elapsed time: 8.683s, Critical Path: 7.58s
```

The compiler processes seem to write to stdout / stderr instead of log files that Bazel allocates. This doesn't work for the intellij plugin or when the process is in background mode.

**Comment by hsyed** [comment]
I'll implement a fix in #54.

**Comment by hsyed** [comment]
fix in PR #54 -- the problem was that the worker infrastructure expects compiler errors to go to stdout.
